src/jit/codegencommon.cpp

```diff
@@ -6304,8 +6304,8 @@ void CodeGen::genZeroInitFrame(int untrLclHi, int untrLclLo, regNumber initReg,
         regMask = genFindLowestBit(availMask);
         rZero2  = genRegNumFromMask(regMask);
         availMask &= ~regMask;
-        assert((genRegMask(rZero2) & intRegState.rsCalleeRegArgMaskLiveIn) ==
-               0); // rZero2 is not a live incoming argument reg
+        assert((genRegMask(rZero2) & intRegState.rsCalleeRegArgMaskLiveIn) == 0); // rZero2 is not a live incoming
```

Comment:
Might be good to move these comments before the assert. Having them at the end of the line is kind of weird, especially when you have line length limits and they're enforced with formatting tools.

Reply:
Yeah, it's weird that jit-format decided to reformat all kinds of lines that I didn't touch.